LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-nonce.html

```diff
@@ -5,7 +5,7 @@
 <script nonce="abc" src="/preload/resources/preload_helper.js"></script>
 <body>
 <script nonce="abc">
-    setup({explicit_done: true});
+    setup({single_test: true});
```

Comment:
Would you be able to add a comment in the commit message explaining why it is necessary to alter the imported test?

Reply:
Updated message.

These changes were already in upstream WPT.